entries/t/trendmicro.com.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Trend Micro": {
+    "domain": "trendmicro.com",
+    "url": "https://www.trendmicro.com",
```

Comment:
If `https://<domain>` redirects to the same page as the `url`-element, then the `url`-element isn't needed.
Since this is the case for this PR, please remove the `url`-element:
```suggestion
```